### PR TITLE
new URLs for the new brand

### PR DESCRIPTION
**Pull request by 0xpulkit**

### CODE REVIEW

File: docs/web-wallet.md [hunk 1]
```diff
@@ -170,61 +170,17 @@ const verificationRequest = {
...
-    chainId: 80002,
-    network: "polygon-amoy",
-  },
-};
-// Encode the verification request to base64
-const base64EncodedVerificationRequest = btoa(JSON.stringify(verificationRequest));
-
-// Open the Polygon ID Verification Web Wallet with the encoded verification request
-window.open(`https://web-wallet-test.polygonid.me/#${base64EncodedVerificationRequest}`);
+window.open(`https://wallet.privado.id/#${base64EncodedVerificationRequest}`);
```
Comment (OBrezhniev) [inline docs/web-wallet.md:173]: Just comment out the onchain part, you would need to bring it back soon.

File: docs/web-wallet.md [hunk 2]
```diff
@@ -150,7 +150,7 @@ const verificationRequest = {
       circuitId: "credentialAtomicQuerySigV2",
       id: 1711399135,
       query: {
-        allowedIssuers: ["*"],
+        allowedIssuers: ["did:polygonid:polygon:amoy:2qV9QXdhXXmN5sKjN1YueMjxgRbnJcEGK2kGpvk3cq"],
```
Comment (OBrezhniev) [inline docs/web-wallet.md:153]: It should be did:iden3:privado:main:.... Ask guys for proper one.

Comment (elias-garcia) [reply docs/web-wallet.md:153]: You can use this for example @0xpulkit:

```
did:iden3:privado:main:2ScrbEuw9jLXMapW3DELXBbDco5EURzJZRN1tYj7L7
```

File: docs/web-wallet.md (hunk 1 above)
Comment (0xpulkit) [reply docs/web-wallet.md:173]: Yes i have commented out the whole on-chain part

File: docs/web-wallet.md (hunk 2 above)
Comment (0xpulkit) [reply docs/web-wallet.md:153]: Sure, Let me ask

File: docs/web-wallet.md (hunk 1 above)
Comment (0xpulkit) [reply docs/web-wallet.md:173]: Will bring it back onec we have it working

File: docs/web-wallet.md (hunk 2 above)
Comment (0xpulkit) [reply docs/web-wallet.md]: have updated the verifier and the issuer did

**Review by OBrezhniev — APPROVED** [review]

**Pull request merged**